bin/kuzzle-createFirstAdmin.js

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env node
+var
+  readlineSync = require('readline-sync'),
+  clc = require('cli-color'),
+  request = require('request-promise'),
+  name,
+  password,
+  step = 0,
+  // steps definitions
+  steps = [
+  	{
+      // get username
+  		prompt: '\n[❓] Which username for the first admin ?\nTip: avoid to name it admin, or root...\n: ',
+  		check: function(userEntry) {
+  			var _name = userEntry.trim(),
+  			badNames = ['admin','root'];
+  			if (badNames.indexOf(_name) === -1) {
+          if (_name.length < 4) {
+            console.log(clc.red('[✖] Username should have at least 4 chars'));
+            return false;
+          }
+
+          name = _name
+  				console.log(clc.green('[✔] Given username: ' + name));
+  				if (_name !== userEntry) {
+  					console.log(clc.cyan('[ℹ] The username have been trimmed.'));
+  				}
+  				name = _name;
+  				return true;
+  			} else {
+  				console.log(clc.red('[✖] It is a bad idea to name an admin "' + _name + '"...'));
+          if (readlineSync.keyInYNStrict('[❓] Are you sure you want to name it "' + _name + '" ?')) {
+            name = _name;
+            return true;
+          }
+  				return false;
+  			}
+  		}
+  	},
+    {
+      // get password
+      password: true,
+      prompt: '\n[?] Which password do you want to set for this account ?\nTips: try to mix lowercased and uppcased letters, digits and special chars. It is always a bad idea to put space char in a password.\n: ',
```

Comment:
Nitpicking: 
- "Enter the account password:"
- "Mix lowercase and uppercase letters, digits and special chars. Avoid using <space> or <tabs> in the password"